### DISCUSSION TITLE
Buildings

### DISCUSSION THREAD

**Discussion by adamhannigan** [sunflower-land/sunflower-land#1159] [Art]
We are looking for 3 new 'building' elements to be used in land expansion for cooking a variety of items. Players will first unlock the open fire, then the oven and then the bakery. Each 'building' will be used to cook different items to feed your characters 👅 

**Requirements**

- Raw minified file
- Screenshot of item on the map
- Base files (aesprite, illustrator etc.) to prove ownership
- Pixel dimensions are 1:1 with the background.

By submitting art you agree to the contributing guidelines and terms/conditions of the project.

**Example**

Below is the example of a Fire Pit which was built to sit in a square of 1x1.

The pixel size matches that of the map perfectly.

<img width="128" alt="Screen Shot 2022-08-16 at 4 49 41 pm" src="https://user-images.githubusercontent.com/11745561/184815445-c2227199-824c-4ee6-8874-621981d15571.png">

**Style**

Please refer to the visual styles here: https://docs.sunflower-land.com/contributing/game-design/visual-and-audio-styles#visual-style

Since these won't be NFTs feel free to extend the base asset pack for your creations.

**Animations**

A non-intrusive animation would be lovely ❤️ We want it to be interesting but not distracting.

**Deadline**

Land Expansion won't go live for at least 3 months but we are looking to finalise these assets by end of September.

Excited to see all the submissions! 🤗 

## Buildings

We will need designs for the following buildings:

### Fire Pit

Dimensions - 1x1 square
Use case - cook food

### Oven / Kitchen

Dimensions - 2x2 squares
Use case - cook food

### Water Well

Dimensions - 1x2 (1 wide, 2 high)
Use case - Support extra crops

### Tent

Dimensions - 3x2
Use case - restore your stamina

Upgrades to House

### Bakery

Dimensions - 3x2 (3 wide, 2 high)
Use case - cook food

**Comment by VadimasVP** [comment]
### Open Fire

1. ![fire](https://user-images.githubusercontent.com/102822587/176935818-f4feb173-0d78-46d5-836d-e3136380a44e.png)

![firebig](https://user-images.githubusercontent.com/102822587/176936046-dddfdf48-b4da-44e3-b2ee-73e2d560d178.png)

![5](https://user-images.githubusercontent.com/102822587/176935853-0badbe1d-79a9-41c4-9f16-d8d740896724.png)


2. ![fire](https://user-images.githubusercontent.com/102822587/176935894-942d52ad-a9b9-4362-a747-83df9d5ffad1.gif)

![firebig](https://user-images.githubusercontent.com/102822587/176936073-a3069406-fc5d-4b0a-8d1a-038e67effa94.gif)

![4](https://user-images.githubusercontent.com/102822587/176935925-6132f6d4-2234-476a-b781-efddeebaeebd.gif)




### Oven

1. ![oven](https://user-images.githubusercontent.com/102822587/176647311-9557471b-c0e9-47f6-a3c6-c170a0c8319d.gif)

![ovenbig](https://user-images.githubusercontent.com/102822587/176647466-6b095d7d-78f3-42cd-a19a-e8704f6200d3.gif)

![1](https://user-images.githubusercontent.com/102822587/176668452-523f9018-18d2-41c4-b59c-3df33e443ea7.gif)



2. ![oven2](https://user-images.githubusercontent.com/102822587/176649002-ddd8ef0c-0963-4076-9e4d-e97f5eb409cc.gif)

![oven2big](https://user-images.githubusercontent.com/102822587/176649343-2123d637-cb4b-4452-97e7-63cd2b7490fb.gif)

![2](https://user-images.githubusercontent.com/102822587/176668474-c7352dd9-d9e5-41e9-8b3f-39948e056aa9.gif)

3. ![oven3](https://user-images.githubusercontent.com/102822587/176684162-24a01b8a-2c71-409e-a36d-8e341081b6f0.gif)

![oven3big](https://user-images.githubusercontent.com/102822587/176684316-c053987d-61da-4007-b5d6-26fc0b0ebae0.gif)


![3](https://user-images.githubusercontent.com/102822587/176684219-dbcf30d2-a77d-48b5-9425-56987d1b8472.gif)





### Bakery

- In progress

**Comment by paluras** [comment]
![1-export](https://user-images.githubusercontent.com/98059798/176667978-da82ecf6-f449-4d0a-b5ba-b90dea4bb800.png)


1:1 Pixel ratio for the background

**Comment by JcEii** [comment]
So here's my work for this art contest. On the open fire art, I drew every frame simply, and I based it on reality (in camping). In the second one, I based it on ancient times where humans used wood to burned it and cook using the ancient oven. I made five versions, which are with or without a door, with or without a fire and oven with smoke. On the last one, I drew a simple bakery, a stall bakery, which is related to the 1900's shop stalls. So I will put the Aseprite files (zipped file) on the last part to prove my ownership.

![all map](https://user-images.githubusercontent.com/105348036/176747437-3517a0ce-f4b9-4cfb-9bd8-d7105fd9bbe8.gif)

### Open Fire
- **Sample**
![open fire](https://user-images.githubusercontent.com/105348036/176747588-42af6a88-63d1-47c1-9705-d46cd3052442.gif)

- **Minified (16x16)**
![Open Fire](https://user-images.githubusercontent.com/105348036/176747884-0257c0ba-7643-4a4e-aa00-5a93f4f08891.gif)

### Oven without door and fire
- **Sample**
![oven1](https://user-images.githubusercontent.com/105348036/176747614-df3eb033-2b6a-4c29-b6e5-f41c1f8f2d5e.png)

- **Minified (18x30)**
![oven without door ](https://user-images.githubusercontent.com/105348036/176747920-f1512f05-abae-40c8-b58f-5aede5bbe955.png)

### Oven with fire without door
- **Sample**
![oven2](https://user-images.githubusercontent.com/105348036/176747643-5b825ff1-9182-4f93-8318-f4cda1ef0290.gif)

- **Minified (18x30)**
![oven without door with fire](https://user-images.githubusercontent.com/105348036/176747951-4056a970-b7f5-43b7-b6c1-4f465adfe2de.gif)

### Oven without fire with door
- **Sample**
![oven3](https://user-images.githubusercontent.com/105348036/176747706-15d369c4-be21-4e54-b344-af01242ea838.png)

- **Minified (18x30)**
![oven with door](https://user-images.githubusercontent.com/105348036/176747976-54f57c0c-25bc-42fb-bf47-8f3c7a9863c6.png)

### Oven with fire and door
- **Sample**
![oven4](https://user-images.githubusercontent.com/105348036/176747814-3541e913-1b75-4515-bec8-5efdc8f577b3.gif)

- **Minified (18x30)**
![oven with door with fire](https://user-images.githubusercontent.com/105348036/176748008-443596f7-f5be-40ac-a2c0-0049e78eece4.gif)

### Oven with fire, door, smoke
- **Sample**
![oven5](https://user-images.githubusercontent.com/105348036/176747831-3932f86d-4ffb-4139-acc8-0c692f9f983c.gif)

- **Minified (18x42)**
![oven with door with fire with smoke](https://user-images.githubusercontent.com/105348036/176748041-0fcdf9d5-d354-47a8-9033-e43b7b90a011.gif)

### Bakery
- **Sample**
![bakery](https://user-images.githubusercontent.com/105348036/176747853-87d9146b-3034-4ddb-839a-9d13cb348e1a.png)

- **Minified (48x48)**
![Bakery](https://user-images.githubusercontent.com/105348036/176748067-5fd85443-5cf8-42dc-aa24-eefe77bb9ca3.png)


### Aseprite file
(old aseprites - black outline)
[Aseprite files art by Jc Eii.zip](https://github.com/sunflower-land/sunflower-land/files/9021934/Aseprite.files.art.by.Jc.Eii.zip)

(latest-edited-aseprites - clean outlines)
[Aseprites uwu.zip](https://github.com/sunflower-land/sunflower-land/files/9022252/Aseprites.uwu.zip)


So that's it thank you.

Discord: Jc Eii #3727
Farm no. 15747

**Comment by FarmerKuenster** [comment]
brick oven
![brick-oven-1](https://user-images.githubusercontent.com/104322749/176803234-a919913f-c2eb-4d7d-ae2d-ff8896d938a2.png)

![brick-oven-1-large](https://user-images.githubusercontent.com/104322749/176803243-dece1222-da6f-4c8e-b1a6-bc8ef3cb9db2.png)

brick oven with animation 
![brick-oven-1-animation](https://user-images.githubusercontent.com/104322749/176803250-938dc174-de50-4046-83b9-a3df8f1cf159.gif)

![brick-oven-1-animation-large](https://user-images.githubusercontent.com/104322749/176803675-3a90a1bc-cd10-4a64-993b-05474c404274.gif)


![brick-oven-1-animation-on-land-large](https://user-images.githubusercontent.com/104322749/176803320-6d8fc259-4d8d-4bbf-a205-b74f53a79655.gif)

discord: farmerkunester #1626
farm id: 32391

**Comment by d4nnY132** [comment]
Hello everyone!
Made only open fire ( in 3 different versions)
![campfire BIG](https://user-images.githubusercontent.com/67920456/176949886-b59519d9-1e4b-47f8-b84a-25c274a6e737.png)
![campfire 2 BIG](https://user-images.githubusercontent.com/67920456/176949903-03af1fcc-1754-410d-a48f-667a08b22dd5.png)
![campfire raw](https://user-images.githubusercontent.com/67920456/176949929-3794fd25-42db-453e-8a47-74985921d765.png)
![campfire 3 BIG](https://user-images.githubusercontent.com/67920456/176949938-5cb8dde0-a81b-4385-9fbb-f004f1a1635b.png)
![campfire2 raw](https://user-images.githubusercontent.com/67920456/176949973-99ffa82c-0da3-45b6-abea-1c03ebe53d6b.png)

**Comment by FarmerKuenster** [comment]
metal oven 
![metal-oven-1](https://user-images.githubusercontent.com/104322749/176980433-91f5ac7b-5bee-44af-b39b-e431d6183720.png)
![metal-oven-1-large](https://user-images.githubusercontent.com/104322749/176980503-1694d577-0bb4-4d9e-9e18-5316a0fa70d8.png)

metal oven with animation 
![metal-oven-1-animation-export](https://user-images.githubusercontent.com/104322749/176980463-d2e01d5f-1ad8-4b8a-b5c4-f8aa2238cffc.gif)
![metal-oven-1-animation-export-2](https://user-images.githubusercontent.com/104322749/176980730-39e6388d-7372-40fe-a6cd-92d6c05d4de7.gif)

![metal-oven-1-animation](https://user-images.githubusercontent.com/104322749/176980452-5d8f4144-2d69-4462-9769-71872268b7db.gif)


discord: farmerkunester #1626
farm id: 32391

**Comment by Kiulg** [comment]
### Open fire (18x28px) ![Fogata28-1](https://user-images.githubusercontent.com/18270876/177192580-5512dc5c-2f09-4608-be67-efd30cfbccbd.gif)
![Fogata28-i](https://user-images.githubusercontent.com/18270876/177194089-f1686641-09a1-4e53-b049-4cba2f93c4e5.gif)



### Open fire (18x18) ![Fogata18-1](https://user-images.githubusercontent.com/18270876/177192693-7c839355-b824-4c06-8896-14341020fc1e.gif)
![Fogata18-i](https://user-images.githubusercontent.com/18270876/177194535-58e7b375-4f64-4b16-bc9b-1f53516d2f40.gif)



### Ashes ![ash1](https://user-images.githubusercontent.com/18270876/177193359-99254b2e-e5da-4f4e-ae39-84f20a781533.png)
![ash4](https://user-images.githubusercontent.com/18270876/177193392-c58a0bbd-30eb-4f51-b652-b95491655d09.png)


### Oven ![Oven1](https://user-images.githubusercontent.com/18270876/177917834-04cce9ab-f895-48d4-bf57-c945d4111537.gif)
![Oven1](https://user-images.githubusercontent.com/18270876/177917757-cf230bac-e966-43f6-862a-e872dd44c377.gif)


### Bakery ![bakery1](https://user-images.githubusercontent.com/18270876/177193082-d4223aaf-f57e-462e-b754-a718f7dd96a7.png)
![bingame](https://user-images.githubusercontent.com/18270876/177194960-0c12d909-b8fd-43c8-b874-80f0a16dfee6.png)

**Reply by VadimasVP** [reply]
Don't you think that your oven is very similar to mine?

**Reply by Kiulg** [reply]
Yes damm you are right, I'd better delete it, sorry

**Comment by esp123** [comment]
firepit
kinda sus animation haha...
![bg-fire](https://user-images.githubusercontent.com/74337946/177079530-665690fb-54af-459b-8890-90c2b88836f8.gif)
![bg-fire1](https://user-images.githubusercontent.com/74337946/177079536-384cdf1f-ba2f-48d3-803d-d8e22befad07.gif)
![bg-firepit1](https://user-images.githubusercontent.com/74337946/177079461-90664564-0710-42c3-8722-9a1f9bee5008.gif)
![bg-firepit](https://user-images.githubusercontent.com/74337946/177079069-0bc45578-ee85-4ee8-bd31-91f4550246cb.gif)

oven
![oven](https://user-images.githubusercontent.com/74337946/177085711-a1c5f9b4-9355-4d55-9cd4-d820cb148068.png)
![oven](https://user-images.githubusercontent.com/74337946/177085964-75c15eb5-124a-4724-976a-91a9d27e6370.gif)
![bg-oven](https://user-images.githubusercontent.com/74337946/177085969-f153f573-3968-47a0-8bee-6bd82c330a2e.gif)

**Comment by eArL2384** [comment]
starting with open fire entry

![Openfireentry1](https://user-images.githubusercontent.com/104702521/177095363-7d239ef2-5b4d-42d4-a508-dcff07864ea8.Gif)

![Openfireentry2](https://user-images.githubusercontent.com/104702521/177095429-0d0ce3cf-6913-4a0a-bc54-e34151a5e9a2.Gif)

![Openfireentry3](https://user-images.githubusercontent.com/104702521/177095478-e358d54b-ba86-44f7-b3d6-2aea754a7c85.Gif)

adding the Kiln Oven at its original size and resized gif
![KilnOvenentry](https://user-images.githubusercontent.com/104702521/177108929-1502f53a-4660-429c-9542-82fd32d026ee.Gif)

![External](https://user-images.githubusercontent.com/104702521/177108973-2d49b7e6-c260-429a-a87d-e0785fe388b4.gif)

Modified the outline for the Kiln Oven

![Externalkiln](https://user-images.githubusercontent.com/104702521/177685038-3619a190-ac33-4892-90d6-c1911ee150c3.gif)

![Externalkilnresized](https://user-images.githubusercontent.com/104702521/177685080-dfefc06f-59e7-4aed-9d51-9b8f07decf61.gif)


Bakery (Kiln was incorporated and was upgraded to Bakery)

![External (5)](https://user-images.githubusercontent.com/104702521/177685187-e107cacc-2522-4406-a891-ceb3284be307.gif)


resized

![ExternalBakeryresize](https://user-images.githubusercontent.com/104702521/177685234-b4d9dc2c-be41-4fc6-bd62-91de0ad0ffd6.gif)

I forgot the raw files 🤣 here they are

![minifiedkiln](https://user-images.githubusercontent.com/104702521/180180260-55fcb2d4-7761-4d9a-b57b-6cc522021ffd.gif)

![minifiedopenfire1](https://user-images.githubusercontent.com/104702521/180180329-fb666b57-b99d-4c4e-8e2c-17e567ad4df8.gif)

![minifiedopenfire2](https://user-images.githubusercontent.com/104702521/180180403-105022a0-92b5-4aeb-aa39-7f35d96aad60.gif)

![minifiedopenfire3](https://user-images.githubusercontent.com/104702521/180180452-fb8415c4-611a-487a-9495-f0e146d8a96c.gif)


Farm ID: #434
Discord ID: eArL2384-ankhknown#6898

**Comment by Vergelsxtn** [comment]
Bakery

preview
![kitchen-preview](https://user-images.githubusercontent.com/99605039/177125671-0e84132f-7719-4548-9732-0efe966faa09.png)
![bakery-assetb](https://user-images.githubusercontent.com/99605039/177125711-861eba1d-c73b-4ab5-b9dd-286f0b233244.gif)
asset
![bakery-asset](https://user-images.githubusercontent.com/99605039/177125771-9e9102fd-1ecf-405f-bf8d-adff63c40d3c.gif)

**Reply by JcEii** [reply]
Wow "May nanalo na" <3<3

**Reply by MrSnorch** [reply]
🧑‍🍳

**Comment by Netherzapdos** [comment]
### Open Fire
---
**Raw File:**
**Without Smoke**
![Open Fire r wO Smoke](https://user-images.githubusercontent.com/101855553/179240082-fbd7f079-16ee-4ff1-b159-638c2820e0d7.gif)

**With Smoke**
![Open Fire r w Smoke](https://user-images.githubusercontent.com/101855553/179240104-7bb6cf1e-f412-4cd2-b2c1-9c42725f7dd8.gif)


**Preview:**
![Open Fire L w Smoke](https://user-images.githubusercontent.com/101855553/179240249-a71c21ee-2477-476b-8a90-5696e7f9198b.gif)
![Open Fire L wo Smoke](https://user-images.githubusercontent.com/101855553/179240252-2c780f28-89b6-4376-a74c-7b98ea860649.gif)


**Raw File for Proof of Ownership:**
[Open Fire R.zip](https://github.com/sunflower-land/sunflower-land/files/9121494/Open.Fire.R.zip)


My reasoning for such a simple looking open-fire is because you're essentially starting from scratch. Something as simple as two sticks, a small fire and a pot seems reasonable for me and really represents your current "progress" in the game, which essentially is back to zero. Also, super happy to have submitted an art piece again despite being super busy, will probs add a fire animation when I have time 😄

**Comment by MrSnorch** [comment]
![печь](https://user-images.githubusercontent.com/104712725/177605294-632997a2-f049-4356-a785-8d87ec13aab3.png)

![печь гиф](https://user-images.githubusercontent.com/104712725/177605331-3ba391cb-4caf-4dda-813b-1dc73fbb813a.gif)

![печь превью-export](https://user-images.githubusercontent.com/104712725/177605833-32e183b8-6fba-4a3a-9fca-34be34a9bfe8.gif)

Discord: **MrSnorch**

**Comment by sfwhat** [comment]
### Open Fire
![openfire](https://user-images.githubusercontent.com/95937808/177629605-0182c753-76c3-4880-aa47-ee0cd9cfc328.gif)
Assets
without ground patch ![2](https://user-images.githubusercontent.com/95937808/177629607-44578858-eeb7-4783-9330-42282697fd8c.gif)
with ![1](https://user-images.githubusercontent.com/95937808/177629611-10572f02-69cb-4150-9c54-b54ef3d10909.gif)


### Oven
![oven](https://user-images.githubusercontent.com/95937808/177641214-c9219fda-93c2-4808-83ea-3f0760076b0c.gif)
Assets
with smoke![oven1](https://user-images.githubusercontent.com/95937808/177641267-dc3f5663-f064-460d-aded-41e8e139c013.gif)
without![oven2](https://user-images.githubusercontent.com/95937808/177641270-037855ab-da60-44a5-b7ad-a5f99938d69a.gif)
Aseprite file if anyone wants to edit: 
[oven.zip](https://github.com/sunflower-land/sunflower-land/files/9057944/oven.zip)


###Bakery
####Gratel and Hansel Style
![Sprite-0002](https://user-images.githubusercontent.com/95937808/181054781-f778e4b2-bc61-4087-bcc0-7ed83c92b935.png)![Sprite-0003](https://user-images.githubusercontent.com/95937808/181054784-c3b47256-e638-41e2-ba6a-b6e6740ea6fa.png)

Assets
![blu](https://user-images.githubusercontent.com/95937808/181054809-3071428f-dfca-4b41-b909-e6457d827663.png)
![pink](https://user-images.githubusercontent.com/95937808/181054812-9f3b550d-6dcc-4a40-9b7c-1d44a289fa89.png)

___
Sfwhat
Farm 6127
Twitter @strugwhat

**Comment by TaRaSiSt** [comment]
**Oven**
![OvenLand](https://user-images.githubusercontent.com/47809767/177770037-c72f25be-2f69-42e4-aa65-40ae941bf2cb.gif) ![Oven2](https://user-images.githubusercontent.com/47809767/178115497-6b1cfcf6-a930-43a0-9817-4a48b5f07cae.gif) ![Oven3](https://user-images.githubusercontent.com/47809767/178785602-6e72d160-5b9f-4ef4-bfd8-8c36faf05e37.gif) ![Oven4](https://user-images.githubusercontent.com/47809767/178845798-eb44a2cc-5b49-4289-a82d-2a59813d943c.gif)




**Assets**
![OvenAsset](https://user-images.githubusercontent.com/47809767/178115514-3798af0c-6c36-4a0a-acf7-1a6269fd7934.png)![Oven2Asset](https://user-images.githubusercontent.com/47809767/178115517-177c4763-de11-45ec-821f-c06d134d56bf.png) ![Oven3](https://user-images.githubusercontent.com/47809767/178785714-b69b90e2-d86f-44bb-8fa6-a5d84342ee73.png) ![Oven4](https://user-images.githubusercontent.com/47809767/178845815-41b58467-86d8-4e70-9270-cb4289cf7e7c.png)

Animation
![OvenAssetFire](https://user-images.githubusercontent.com/47809767/177771122-ab966d1b-dffa-45a7-9742-5a9bf4029ba9.gif) ![Oven2fire](https://user-images.githubusercontent.com/47809767/178115541-24a97557-21c4-4722-9a36-b0b3946f6b92.gif) ![Oven2smoke](https://user-images.githubusercontent.com/47809767/178115547-8d23bb81-e8f4-444f-b92b-2fdf9b152b5a.gif) ![Oven3Anim](https://user-images.githubusercontent.com/47809767/178785741-a5da47d1-c737-4a19-bd76-d7787ea4b130.gif) ![Oven4Anim](https://user-images.githubusercontent.com/47809767/178845826-318641be-2e1b-45f1-b55c-049963c78d27.gif)




**SFL Bakery** 
![BakeryV2 0Big](https://user-images.githubusercontent.com/47809767/177884110-2add8cf5-98a2-4197-8d1f-f6fe6b19df2a.gif) ![BakeryV3 0](https://user-images.githubusercontent.com/47809767/178261304-bbd2d891-d5d4-4926-b549-e6e22cb1dc13.gif) ![BakeryV4 0](https://user-images.githubusercontent.com/47809767/178331758-2d10aba3-7e6c-47a5-a2ed-2d44356b3db8.gif)


1x1 px
![BakeryV2 0](https://user-images.githubusercontent.com/47809767/177884241-39045971-8a99-4f24-9cec-61f5af0fd3bb.gif) ![BakeryV3 0-1x1](https://user-images.githubusercontent.com/47809767/178261333-4e2b99a4-8655-4573-a16b-7aad4e621042.gif) ![BakeryV4 0-1x1](https://user-images.githubusercontent.com/47809767/178331785-cc1f3644-9e91-4d3d-a448-9f5a0a94e92b.gif)


**Assets**
![BakeryV2 0AssetNoFire](https://user-images.githubusercontent.com/47809767/177884170-01922ac9-f4e1-493e-8b7a-112c5cfad7ba.png) ![BakeryV3Asset 0](https://user-images.githubusercontent.com/47809767/178261363-48aab8a4-dd61-48e3-a11a-913191d93e63.png) ![BakeryV4 0Asset](https://user-images.githubusercontent.com/47809767/178331813-96119392-ad7f-4351-9c5b-1b63a5e1d0b9.png)


Animation
![BakeryV2 0AssetFire](https://user-images.githubusercontent.com/47809767/177884180-b5fb7c8e-262d-4dc6-9a7a-a8ac1f0359ea.gif) ![BakeryV3 0AssetAnim](https://user-images.githubusercontent.com/47809767/178331679-d197939e-77f0-405c-bceb-9cd650876260.gif) ![BakeryV4 0AssetAnim](https://user-images.githubusercontent.com/47809767/178331848-f34f6334-0b81-4d3e-9797-ca0dcfa33cfe.gif)

Discord **TaRaSiSt#7782** ^_^
Farm id 28877

**Reply by Q4FOIL** [reply]
I love it ❤️

**Comment by FarmerKuenster** [comment]
Bakery ideas! 

Without oven attached
![bakery1](https://user-images.githubusercontent.com/104322749/178081394-8817511f-1c17-4d8c-a4c0-94fd2635b222.png)
![bakery1-large](https://user-images.githubusercontent.com/104322749/178081401-31fed4dc-c8ec-4330-ab0b-71bced441d25.png)
![bakery-on-land-export](https://user-images.githubusercontent.com/104322749/178081442-42437983-3d3d-4a63-98cf-1d703b12b7c7.png)

With oven attached 
![bakery1-with-oven](https://user-images.githubusercontent.com/104322749/178085487-4e35d756-3ca0-4061-aa2f-12d6187daffd.png)
![bakery1-with-oven-large](https://user-images.githubusercontent.com/104322749/178085491-6fdc8474-0e74-4e05-84aa-e2a2169e01ba.png)
![bakery-with-oven-on-land](https://user-images.githubusercontent.com/104322749/178085481-d20e679e-1e9e-4451-862f-50f6820ba1e2.png)

oven with animation added
![bakery1-with-oven-animation](https://user-images.githubusercontent.com/104322749/179633277-32b11a95-add1-46fa-8a59-c3561b3ca8da.gif)
![bakery1-with-oven-animation-LARGE-export](https://user-images.githubusercontent.com/104322749/179633203-57ca79c5-ae8a-4bde-8b18-11b065ceb534.gif)
![bakery-on-land-LARGE](https://user-images.githubusercontent.com/104322749/179633140-7d418f75-70ff-46c4-b1f2-81a8acb57a5f.gif)

discord: farmerkunester #1626
farm id: 32391

**Comment by datorymen** [comment]
Hope I am not too late.... :)

**Open Fire**

Raw minified file
![Open fire](https://user-images.githubusercontent.com/87224912/183257852-555d1ec6-f629-4609-bfce-8acd8fe8e2af.gif)

Screenshot of item on the map
![Open fire submission](https://user-images.githubusercontent.com/87224912/183257863-15ef339b-4819-4ff2-a23f-b2570e1e3f65.gif)

Base files (aesprite, illustrator etc.) to prove ownership
[datory.men Open Fire Aseprite files.zip](https://github.com/sunflower-land/sunflower-land/files/9275430/datory.men.Open.Fire.Aseprite.files.zip)

Farm ID: 143148
Discord ID: datory.men#9568
